src/connector/src/sink/remote.rs

```diff
@@ -149,11 +149,14 @@ impl<R: RemoteSinkTrait> Sink for RemoteSink<R> {
     }
 
     async fn validate(&self) -> Result<()> {
-        validate_remote_sink(&self.param).await
+        validate_remote_sink(&self.param)
+            .await
+            .context("failed to validate remote sink")?;
```

Comment:
Is `src/meta/src/stream/sink.rs` the only entrance? If so, then no need to `context` here. 🤔